config/mg.py

```diff
@@ -20,7 +20,14 @@
     "",
     byte.BYTES.closure())
 
-LANGUAGE_SPECIFIC_PREPROCESSING = MG_VELAR_NASAL
+MG_ABBREVIATION = cdrewrite(
+    cross("@", "amin'ny"),
+    union("[BOS]", byte.SPACE),
+    union("[EOS]", byte.SPACE),
+    byte.BYTES.closure())
+
+#LANGUAGE_SPECIFIC_PREPROCESSING = MG_VELAR_NASAL
```

Comment:
remove this comment?

Reply:
Malagasy currently has two language-specific rules (the velar nasal one and the abbreviation one). Do we want to experiment with each of them individually, or will we want to combine them and do one experiment for normalizing Malagasy? If the latter, we can delete the comment but change the next line to 
`LANGUAGE_SPECIFIC_PREPROCESSING = MG_VELAR_NASAL @ MG_ABBREVIATION`.

Reply:
Oh I see, yeah, we can definitely experiment with adding both fixes at the same time :).

Reply:
Done!